### ISSUE TITLE
Add Rachel to website

### ISSUE THREAD

**Issue by Nikomus** [neontribe/www#430]
## Description
Add Rachel to The Tribe page

## Plan
- [ ] add three words
- [ ] add description
- [ ] add image
- [ ] add any social media link

## Acceptance Criteria  
- [ ] On The Tribe page, I can see Rachel's profile with image, three words, description and any social links

**Comment by rachelpeharding** [comment]
3 (4) words:   
Creativity, Learning, Outdoorsy Stuff  

Description: 
  On the cusp of beginning a PhD in Psychology, I realised I needed to problem solve in a more visually creative way. Through a couple of intense years dedicating myself to the creative process, I found my passion working in design. At Neontribe, I enjoy the challenge of designing for function and communication, whilst maintaining an artistic look and feel in my work. I deeply care about working on projects for good and hope that I am helping people in a small but meaningful way. Outside of work I can be found climbing, drawing or exploring nature.
